dropwizard-jdbi3/src/main/java/io/dropwizard/jdbi3/NamePrependingTemplateEngine.java

```diff
@@ -0,0 +1,46 @@
+package io.dropwizard.jdbi3;
+
+import org.jdbi.v3.core.extension.ExtensionMethod;
+import org.jdbi.v3.core.statement.StatementContext;
+import org.jdbi.v3.core.statement.TemplateEngine;
+
+import java.util.Objects;
+
+/**
+ * Jdbi {@link TemplateEngine} that prepends the SQLObject's type and method name in front of every
+ * statement sent to the database.
+ */
+public class NamePrependingTemplateEngine implements TemplateEngine {
```

Comment:
It would be nice to have a simple unit test for `NamePrependingTemplateEngine` to verify that it actually inserts comments to the query.

Reply:
Test added